src/app/indexedDB/facility-db.service.ts

```diff
@@ -125,6 +125,13 @@ export class FacilitydbService {
         });
     }
 
+    deleteAllAccountFacilities(): void {
+        let pendingFacilities = JSON.parse(JSON.stringify(this.accountFacilities.value));
+        for(let i=0; i<pendingFacilities.length; i++) {
+            this.dbService.delete('facilities', pendingFacilities[i]['id']);
```

Comment:
Nit picking.. can we use .id instead of ['id']. Helps compiler inform us if we messed something up down the road if we ever refactor that variable.

I see this is in the other services as well. Just gonna use this one comment for all of those